### PR TITLE
[FIX] account: fix bank statement reconciliation writeoff creation

### PR DESCRIPTION
**Pull request by csnauwaert**
Use case:
Company is in USD, create a bank journal in euro and set euro currency to 0.9
Create invoice for 80 EUR and pay it. Then create a bank statement of 85 EUR in newly created journal and use reconciliation widget to reconcile statement and payment. Put the remaining 5 EUR as writeoff in bank fees. The move created for those 5 EUR wrongly compute the amount_currency of the counterpart, this commit fix this problem.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by qdp-odoo** [comment]
runbot is red

**Comment by csnauwaert** [comment]
@qdp-odoo should be green after those changes

File: addons/account/tests/test_reconciliation.py [hunk 1]
```diff
@@ -406,3 +406,56 @@ def create_move(name, amount, amount_currency, currency_id):
...
+            'debit': 0,
+            'credit': 5,
+            'name': 'bank fees',
+            }])
+
+        # Check that move lines associated to bank_statement are correct
+        self.check_results(bank_stmt.move_line_ids, {
+            self.account_usd.id: {'debit': 3.27, 'credit': 0.0, 'amount_currency': 5, 'currency_id': self.currency_usd_id},
+            self.diff_income_account.id: {'debit': 0.0, 'credit': 3.27, 'amount_currency': -5, 'currency_id': self.currency_usd_id},
+            self.account_rcv.id: {'debit': 0.0, 'credit': 52.33, 'amount_currency': -80, 'currency_id': self.currency_usd_id},
```
Comment (qdp-odoo) [inline addons/account/tests/test_reconciliation.py]: I think you miss the line
+            self.account_usd.id: {'debit': 52.33, 'credit': 0, 'amount_currency': 80, 'currency_id': self.currency_usd_id},

no?

**Comment by csnauwaert** [comment]
merged at rev: fe40560c31ce69eaa80279dfba313846522986cb

**Pull request closed without merging**